src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2039,36 +2040,38 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
 
   // Values calculated using torch.nn.BatchNorm2d(momentum = None).
   output = model.TrainingMean();
-  result << 3.33333333 << arma::endr
-         << 3.1 << arma::endr
-         << 3.06666666 << arma::endr;
+  
+result = { { 3.33333333 },
+           { 3.1 },
+           { 3.06666666 } };
 
   CheckMatrices(output, result, 1e-1);
 
   // Values calculated using torch.nn.BatchNorm2d().
   output = model2.TrainingMean();
-  result << 0.3333 << arma::endr
-         << 0.3100 << arma::endr
-         << 0.3067 << arma::endr;
-
+  
+result = { { 0.3333 },
```

Comment:
```suggestion
  result = { { 0.3333 },
```
This initialisation and a few others are not indented properly.

Reply:
thanks @Abilityguy  .I will resolve the problem soon.